Client/Frontend/Theme/DarkTheme.swift

```diff
@@ -4,19 +4,150 @@
 
 import Foundation
 
+// Convenience reference to these normal mode colors which are used in a few color classes.
+fileprivate let defaultBackground = UIColor.Photon.Ink80
+fileprivate let defaultSeparator = UIColor.Photon.Grey30
+fileprivate let defaultTextAndTint = UIColor.Photon.Grey10
+
+fileprivate class DarkTableViewColor: TableViewColor {
+    override var rowBackground: UIColor { return UIColor.Photon.Ink70 }
+    override var rowText: UIColor { return defaultTextAndTint }
+    override var disabledRowText: UIColor { return UIColor.Photon.Grey40 }
+    override var separator: UIColor { return UIColor.Photon.Grey60 }
+    override var headerBackground: UIColor { return UIColor.Photon.Ink80 }
+    override var headerText: UIColor { return UIColor.Photon.Grey30 }
+//    override var rowActionAccessory: UIColor { return UIColor.Photon.Blue50 }
+//    override var controlTint: UIColor { return rowActionAccessory }
+//    override var syncText: UIColor { return defaultTextAndTint }
+//    override var errorText: UIColor { return UIColor.Photon.Red50 }
+//    override var warningText: UIColor { return UIColor.Photon.Orange50 }
+}
+
+fileprivate class DarkActionMenuColor: ActionMenuColor {
+    override var foreground: UIColor { return defaultTextAndTint }
+    override var iPhoneBackground: UIColor { return UIColor.Photon.Ink90.withAlphaComponent(0.8) }
```

Comment:
Do we not have an `Ink90A80`? If so we should use that, if not, we should talk to the DSG team about adding one…

Reply:
This is not the correct color. There is no way to specify an exact color with the iOS blur effect, so I am not sure what we plan for this. Their blur algorithm has low color preservation.